bricks/next-builder/src/search-tree/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {
+  BrickWrapper,
+  UpdatingElement,
+  property,
+  event,
+  EventEmitter,
+} from "@next-core/brick-kit";
+import { SearchTree } from "./SearchTree";
+import { StoryboardAssemblyResult } from "../shared/storyboard/interfaces";
+
+/**
+ * @id next-builder.search-tree
+ * @author SailorF
+ * @history
+ * 1.x.0: 新增构件 `next-builder.search-tree`
+ * @docKind brick
+ * @noInheritDoc
+ */
+export class SearchTreeElement extends UpdatingElement {
+  connectedCallback(): void {
+    // Don't override user's style settings.
+    // istanbul ignore else
+    if (!this.style.display) {
+      this.style.display = "block";
+    }
+    this._render();
+  }
+
+  /**
+   * @kind string
+   * @required true
+   * @default -
+   * @description 指定的应用 homepage
+   */
+  @property()
+  homepage: string;
+
+  /**
+   * @kind string
+   * @required true
+   * @default -
+   * @description 指定的应用 appId
+   */
+  @property()
+  appId: string;
+
+  /**
+   * @kind string
+   * @required true
+   * @default -
+   * @description 指定的应用 projectId
+   */
+  @property()
+  projectId: string;
+
+  /**
+   * @default -
+   * @required true
+   * @description 搜索树数据
+   */
+  @property()
```

Comment:
默认为 `{ type: String }`，其它支持的类型有 `Number` `Boolean`，其它复合类型的属性需要写：

```suggestion
  @property({ attribute: false })
```

底层的原因是简单类型的 properties 可以在元素的 attributes 上映射，而复合类型的不能。